log/src/main/java/br/com/codenation/log/service/interfaces/UsuarioServiceInterface.java

```diff
@@ -0,0 +1,10 @@
+package br.com.codenation.log.service.interfaces;
+
+import br.com.codenation.log.entity.Usuario;
+
+import java.util.Optional;
+
+public interface UsuarioServiceInterface {
+    Optional<Usuario> buscaPorEmail(String email);
+    Usuario salvar(Usuario usuario);
```

Comment:
```suggestion
    Optional<Usuario> salvar(Usuario usuario);
```
Poderia padronizar usando Optional em ambos métodos.

Reply:
mesmo o método save do repository nunca retornando null ainda assim seria melhor usar optional para manter o padrão? Nesse caso eu usaria um try catch e retornaria um Optional vazio quando ocorrer algum problema no salvamento?